### ISSUE TITLE
[Question]: Example of get_activities()

### ISSUE THREAD

**Issue by sshcli** [alpacahq/alpaca-py#192]
### Question form pre-submit checklist.

- [X] I have searched the existing issues to ensure there isn't already an issue about this question.
- [X] My question has to do with the Python SDK and isn't a general question about the API. (If it is please open your issue [here](https://github.com/alpacahq/Alpaca-API))
- [X] My question isn't about how to do a specific algorithm or asking for trade advice (answers to these are outside the scope of this repo).

### Question

There is no example in documentation about how to get stock activities in the new official SDK

In `alpaca-trade-api version: 2.3.0` is as simple as: `get_activities()`

Please add an example in the docs and close this issue with the reference URL.

Thanks in advance.

Regards,

**Comment by brettelliot** [comment]
Agreed. Would be nice to have alpaca-py support account activities.

**Comment by sshcli** [comment]
A feature request has been created for this.
See: https://github.com/alpacahq/alpaca-py/issues/595
